### PR TITLE
Update button component

### PR DESCRIPTION
**Pull request by ileenshaar**

### CODE REVIEW

File: src/components/carousel/components/carousel-item/carousel-item.jsx [hunk 1]
```diff
@@ -23,20 +23,39 @@ export function CarouselItem({ title, description, imgPath }) {
...
+                <path
+                  d="M1 5.49996H15M15 5.49996L10.5 1M15 5.49996L10.5 9.99996"
+                  stroke="white"
+                  stroke-width="2"
+                  stroke-linecap="round"
+                  stroke-linejoin="round"
+                />
+              </svg>
+            }
+            label={"See More"}
```
Comment (mahmoud-alragabi) [inline src/components/carousel/components/carousel-item/carousel-item.jsx]: the text should be passed as prop

File: src/components/carousel/components/carousel-item/carousel-item.jsx [hunk 2]
```diff
@@ -23,20 +23,39 @@ export function CarouselItem({ title, description, imgPath }) {
...
+          <CategoryTitles title={title} description={description} />
+        </Box>
 
-          <Box sx={{ position: "absolute", left: "42%", top: "70%" }}>
-            <IconicButton icon={<EastIcon />} label={"See More"} />
-          </Box>
-        </Link>
+        <Box sx={{ position: "absolute", left: "42%", top: "70%" }}>
+          <IconicButton
+            icon={
```
Comment (mahmoud-alragabi) [inline src/components/carousel/components/carousel-item/carousel-item.jsx]: icon should be passed as prop

**Review by mahmoud-alragabi — CHANGES_REQUESTED** [review]

**Review by mahmoud-alragabi — CHANGES_REQUESTED** [review]

File: src/components/carousel/components/carousel-item/carousel-item.jsx [hunk 3]
```diff
@@ -34,7 +33,27 @@ export function CarouselItem({ title, description, imgPath }) {
...
+                  fill="none"
+                >
+                  <path
+                    d="M1 5.49996H15M15 5.49996L10.5 1M15 5.49996L10.5 9.99996"
+                    stroke="white"
+                    stroke-width="2"
+                    stroke-linecap="round"
+                    stroke-linejoin="round"
+                  />
+                </svg>
```
Comment (mahmoud-alragabi) [inline src/components/carousel/components/carousel-item/carousel-item.jsx:52]: replace this with a component for the icon, that's cleaner and more readable

**Review by mahmoud-alragabi — CHANGES_REQUESTED** [review]

**Review by mahmoud-alragabi — APPROVED** [review]

File: src/components/carousel/carousel.jsx
Comment (mahmoud-alragabi) [inline src/components/carousel/carousel.jsx:1]: remove this commit

**Review by mahmoud-alragabi — CHANGES_REQUESTED** [review]

**Review by mahmoud-alragabi — APPROVED** [review]

**Review by ferassmohana — APPROVED** [review]

**Pull request merged**